hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/TarContainerPacker.java

```diff
@@ -213,6 +221,31 @@ private byte[] readEntry(InputStream input, final long size)
     return output.toByteArray();
   }
 
+  private void includeMetadataFiles(KeyValueContainerData containerData,
+      ArchiveOutputStream archiveOutput) throws IOException {
+    if (containerData.getSchemaVersion().equals(OzoneConsts.SCHEMA_V3)) {
+      includeDumpFiles(Paths.get(containerData.getMetadataPath()),
```

Comment:
Basically we can reuse the includePath here. There should be only dumped table files in this directory.

Reply:
Continue the last comment here.
We have a concrete on-disk db directory under the `metadata` directory(like: metadata/1-dn-container.db) for per-container db instance, but we don't have one dedicated for temp sst files, for now we just put it alongside with the container meta file(like: metadata/{1-container, block_data.data, metadata.data, ...}).
So I intend to include exactly the files that match the suffix here.

Reply:
But...maybe we could create a temp dir while exporting like "metadata/db" or other names, and reuse includePath?

Reply:
It's a good idea.